R/pkg/R/DataFrame.R

```diff
@@ -1187,6 +1187,65 @@ setMethod("dapply",
             dataFrame(sdf)
           })
 
+#' gapply
+#'
+#' Apply a function to each group of a DataFrame.
```

Comment:
In the description, we need to explain what is a group; Otherwise, users will not know how to use it.